lib/helper.js

```diff
@@ -53,20 +55,13 @@ const isActive = (player, state, rc) => {
   if (!lastRec || !lastRec.type || !lastRec.amount) {
     // in waiting or dealing a player might not have a receipt yet
     if (state === 'waiting' || state === 'dealing') {
-      if (player.sitout) {
-        return false;
-      }
       return true;
     }
-    if (player.sitout) {
-      return false;
-    }
     throw new Error('receipt undefined');
   } else {
     // check receipt type for receipts we don't want
     if (lastRec.type === Type.FOLD
       || lastRec.type === Type.SHOW
-      || player.sitout
```

Comment:
Removing `player.sitout` conditions change function's behavior, isn't?

Reply:
this condition will never happen, because if `player.sitout` is true, `isActive` returns false at https://github.com/acebusters/poker-helper/pull/160/files/96edcf3ca8e4da0da0f3b8eb016ecfc6f427081e#diff-edd552e5b38c38c77f6df11eb7b22ebeL48

Reply:
Missed this line 🙃